src/master-playlist-controller.js

```diff
@@ -256,7 +262,22 @@ export class MasterPlaylistController extends videojs.EventTarget {
     // combined audio/video or just video when alternate audio track is selected
     this.mainSegmentLoader_ = new SegmentLoader(segmentLoaderOptions);
     // alternate audio track
+    segmentLoaderOptions.loaderType = 'audio';
     this.audioSegmentLoader_ = new SegmentLoader(segmentLoaderOptions);
+
+    this.decrypter_.onmessage = (event) => {
+      switch (event.data.source) {
```

Comment:
Matt and I discussed turning this into a set of if/else instead of an *evil* switch statement!! Just noting that incase someone else notices the switch statement and feels ill.